email/1.5/modules/ROOT/pages/email-troubleshooting.adoc

```diff
@@ -0,0 +1,239 @@
++
+[source,plain-text]
+----
+DEBUG 2021-04-26 18:51:08,536 [_pollingSource_imapattachmentFlow/executor.01] [processor: ; event: ] org.mule.extension.email.internal.mailbox.BaseMailboxPollingSource: Poll will be skipped, since last poll emails are still being processed
+INFO  2021-04-26 18:51:09,166 [[MuleRuntime].uber.04: [imapattachment].imapattachmentFlow.CPU_LITE @111719e0] [processor: imapattachmentFlow/processors/1/processors/2; event: 820f7fe0-a6d9-11eb-a84b-147dda4dba09] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: "" as Binary {base: "64"}
+DEBUG 2021-04-26 18:51:15,360 [_pollingSource_imapattachmentFlow/executor.01] [processor: ; event: ] org.mule.extension.email.internal.mailbox.BaseMailboxPollingSource: Email [172] was not processed.
+....
+----
+
+== Troubleshoot Parsing Text File Attachment as Body Message
+
+When Email Connector receives a message without body and only an attachment file with Content-Type as `text`, the connector publishes the text content as payload body. To receive the text file in the payload attachments list, set the `parsing.text.attachment.as.body` system property to `false`. Refer to the xref:email-attachmen.adoc[set parsing text file attachment as body message] documentation for details.
+
+== Troubleshoot Jakarta Mail System Properties
+
+Email Connector uses the Java Mail library. Thereby, you can change certain aspects how the connector behaves by changing specific Jakarta Mail system properties, such as MIME type and MIME multipart properties. These properties control conformance to the MIME type specification.
+
+Note that these properties are not session properties and you must set them globally as system properties. To learn how to set system properties in Anypoint Studio or for on-premises deployments, refer to the xref:mule-runtime::mule-app-properties-system.adoc[system properties] documentation.
+
+=== MIME Type System Properties
+
+These are the most consulted Jakarta Mail MIME type system properties:
+
+* `mail.mime.decodetext.strict` +
+Controls decoding of MIME encoded-words. The MIME specification requires that encoded-words start at the beginning of a whitespace-separated word. Some mailers incorrectly include encoded-words in the middle of a word. If the `mail.mime.decodetext.strict` system property is set to `false`, an attempt is made to decode these illegal encoded-words. Defaults to `true`.
+
+* `mail.mime.encodeeol.strict` +
+Controls the choice of Content-Transfer-Encoding for MIME parts that are not of type `text`. Often such parts contain textual data for which an encoding that allows the normal end of line conventions is appropriate. In rare cases, such a part appears to contain entirely textual data but requires an encoding that preserves CR and LF characters without change. If the `mail.mime.encodeeol.strict` system property is set to `true`, such an encoding is used when necessary. Defaults to `false`.
```

Comment:
Controls the choice of Content-Transfer-Encoding for MIME parts that are not of type `text`. Often these MIME parts contain textual data for which an encoding that allows the normal end of line conventions is appropriate. In rare cases, MIME parts that are not of type `text`  appear to contain entirely textual data, but they require an encoding that preserves the CR and LF characters without change. If the `mail.mime.encodeeol.strict` system property is set to `true`, this type of encoding is used when necessary. Defaults to `false`.